# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsHttpChannel.h
# Commit: bc2b36de8f33
# Full Hash: bc2b36de8f33a7705e0c672987128159f8f4427c
# Author: Paul Bone <pbone@mozilla.com>
# Date: 2020-03-18 04:14:11
# Description:
#   Bug 1611907 - Remove AssertNotDocumentChannel r=mattwoodrow
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D67093
# ==============================================================================

diff -r 02ef6cee073c -r bc2b36de8f33 netwerk/protocol/http/nsHttpChannel.h
--- a/netwerk/protocol/http/nsHttpChannel.h	Tue Mar 17 23:38:28 2020 +0000
+++ b/netwerk/protocol/http/nsHttpChannel.h	Tue Mar 17 19:20:58 2020 +0000
@@ -350,9 +350,6 @@
   MOZ_MUST_USE nsresult ContinueProcessResponse1();
   MOZ_MUST_USE nsresult ContinueProcessResponse2(nsresult);
 
- private:
-  void AssertNotDocumentChannel();
-
  public:
   void UpdateCacheDisposition(bool aSuccessfulReval, bool aPartialContentUsed);
   MOZ_MUST_USE nsresult ContinueProcessResponse3(nsresult);
